CHANGELOG.md

```diff
@@ -5,11 +5,25 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
+## [0.5.0] - 2015-12-22
+### Changed
+- updated the way makefiles are generated
```

Comment:
I'm not wedded to my copy; we could probably pull verbiage from @adrienthebo's PR and commits on this. But it's probably the single biggest backend change and the one most likely to expose edge cases, so I'd want to make sure it's called out/spelled out for users.